e2e/node_bootstrapper_test.go

```diff
@@ -51,7 +51,7 @@ func Test_ubuntu2204NodeBootstrapper(t *testing.T) {
 			LiveVMValidators: []*LiveVMValidator{
 				mobyComponentVersionValidator("containerd", getExpectedPackageVersions("containerd", "ubuntu", "r2204")[0], "apt"),
 				mobyComponentVersionValidator("runc", getExpectedPackageVersions("runc", "ubuntu", "r2204")[0], "apt"),
-				FileHasContentsValidator("/var/log/azure/node-bootstrapper.log", "node-bootstrapper finished successfully"),
+				FileHasContentsValidator("/var/log/azure/node-bootstrapper.log", "node-bootstrapper finished"),
```

Comment:
did you finally managed to run it?

Reply:
no :(  i just noticed the validator from the other test was failing because the string in the binary changed